clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/ops/DeployAppengineAtomicOperation.groovy

```diff
@@ -117,9 +122,37 @@ class DeployAppengineAtomicOperation implements AtomicOperation<DeploymentResult
 
     task.updateStatus BASE_PHASE, "Deploying version $versionName..."
     jobExecutor.runCommand(deployCommand)
+    deleteFiles(writtenFullConfigFilePaths)
```

Comment:
Yes to the first point - good suggestion.

I just verified that it is not asynchronous, so the deletion should be fine.